stories/button.stories.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import Button from '../src/web/components/Button';
+import Icon from '../src/web/components/Icon';
+
+storiesOf('Button', module)
+  .add('basic button', () => <Button onClick={action('button-click')}>Simple button</Button>)
+  .add('with size', () => [
+    <Button onClick={action('button-click1')} size="small">
```

Comment:
Instead of `button-clikc1`, is this possible to put the size, like `'small'` ?

Reply:
Or the color ? (Or the discriminent)

Reply:
Ok, i will do thaht